free-1.md

```diff
@@ -1,43 +1,53 @@
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I once thought audio mixing would be an extremely easy thing to do. I had several ideas, but was met with more barriers to entry than I expected, including both the cost and complexity of existing software. Although these tools have been proven to be the best for the trade, these barriers alone were enough to make me abandon the idea. I felt like at the time there existed two extremely polarized types of audio mixing software: extremely powerful but extremely complicated, and extremely simple but extremely easy. I'm hoping this DSL will find somewhat of a middle ground. 
 
 ### Domain
 _Describe the project's domain in five words._
 
+Music mixing for the amateur
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The program would take in any number of audio files and allow the user to apply some basic audio manipulation tools to it such as volume, bass, speed adjustments, cutting and looping tools, simultaneously playing multiple tracks, among others (mostly limited by the scope of the project). The user could combine some of these tools to create their own. Programming might look something like LateX in that it would look very structured with sequential blocks that correspond to parts of the music file (e.g. \begin{volume-boost}@3:44 ... \end{volume-boost}@3:55) or something along those lines. I think this is a correct way to approach the problem as it makes the structure of the music files more apparent without any visual ui. 
+
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+The program would run through the code above when a specific key word is found (e.g. play at the bottom of the file) and apply any effects to the song. A program interacts with the user by outputting any changes they make to the song. Some errors that might occur include applying an undefined effect, applying an effect outside the bounds of the current song, undefined behavior when combining effects, among others. 
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to do very basic individual edits like increasing volume at a certain portion of the song or increasing the bass near the end. It should be possible to actually make a song from scratch using several samples created from another audio recording program, but this would be extremely difficult to do. This language seems more for editting an existing song. It would be very difficult, if not impossible, for users to make production quality music with this, I imagine, given its rather basic tools. 
 
```

Comment:
If you're trying to make something to edit one song, combine multiple songs, and let users make their own songs, I'd be careful about the scope of the project. Think about how you can reuse the same logic for all of those tasks.